src/node/ccf_acme_client.h

```diff
@@ -31,21 +39,133 @@ namespace ccf
     }
   }
 
+  class ACMERpcEndpoints : public CommonEndpointRegistry
```

Comment:
Could you move the server implementation to another file to keep this separate from the client logic?